doc/manifest/schema/1.1.0/defaultLocale.md

```diff
@@ -0,0 +1,239 @@
+
+  >Note: This field is the key to determining which fields are required for the Microsoft community repository. The default locale specified in the version file must match with this value.
+
+ </details>
+
+#### Publisher
+
+<details>
+  <summary>The publisher name</summary>
+
+  **Required Field**
+  This key represents the name of the publisher for a given package. This field is intended to allow the full publisher's or ISV's name to be displayed as they wish.
+
+  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 and Windows 11 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+ </details>
+
+#### PublisherUrl
+
+<details>
+  <summary>The publisher home page</summary>
+
+  **Optional Field**
+  This key represents the web site for the publisher or ISV.
+ </details>
+
+#### PublisherSupportUrl
+
+<details>
+  <summary>The publisher support page</summary>
+
+  **Optional Field**
+  This key represents the customer support web site or specific web page provided by the publisher or ISV.
+ </details>
+
+#### PrivacyUrl
+
+<details>
+  <summary>The publisher privacy page or the package privacy page</summary>
+
+  **Optional Field**
+  This key represents the privacy web site or specific web page provided the publisher or ISV. If there is a privacy web site or specific web page for the package it is preferred over a generic privacy page for the publisher.
+ </details>
+
+#### Author
+
+<details>
+  <summary>The package author</summary>
+
+  **Optional Field**
+  Thie key represents the author of a package. In some cases, the author is an individual who develops and or maintains the package.
+ </details>
+
+#### PackageName
+
+<details>
+  <summary>The package name</summary>
+
+  **Required Field**
+  This key represents the name of the package. This field is intended to allow the full package name to be displayed as the publisher or ISV wishes.
+
+  >Note: With the 1.0 release of the Windows Package Manager, this name affects how packages from a source are mapped to Apps installed in Windows 10 via Add / Remove Programs (ARP). The best practice is to ensure this matches the ARP entry for the package name when it has been installed. The impact is associated with `winget upgrade` and `winget list`.
+ </details>
+
+#### PackageUrl
+
+<details>
+  <summary>The package home page</summary>
+
+  **Optional Field**
+  This key represents the web site for the package.
+ </details>
+
+#### License
+
+<details>
+  <summary>The package license</summary>
+
+  **Required Field**
+  This key represents the license governing the use and or distribution for the product. This could be an open source license, or a commercial license.
```

Comment:
```suggestion
  This key represents the license governing the use and or distribution for the product. This could be an open source license, or a commercial license. Please note that a copyright is not considered a license. If there is no available information on a product's license, "Proprietary" should be the value in this field.
```